File: testing/demo/plugins/src/pydemo/proto/pb.py
def onReadySending(m, c, r: dict):
    print("onReadySending", m, c, r)


def onReceived(m, c, r: dict):
    print("onReceived", m, c, r)


def onError(m, c, r: dict):
    print("onError", m, c, r)

File: testing/demo/plugins/src/demo/pb/pb.go
package pb

import "fmt"

func OnReadySending(m, c, r map[string][]byte) {
	fmt.Println("OnReadySending message templates", m, "config", c, "request attributes", r)
}

func OnReceived(m, c, r map[string][]byte) {
	fmt.Println("OnReceived message templates", m, "config", c, "response", r)
}

func OnError(m, c, r map[string][]byte) {
	fmt.Println("OnError message templates", m, "config", c, "response", r)
}
